File: example04retrofit/src/main/java/foo/bar/example/foreretrofit/ui/fruit/FruitFragment.java
package foo.bar.example.foreretrofit.ui.fruit;

import android.os.Bundle;
import android.support.v4.app.Fragment;
import android.view.LayoutInflater;
import android.view.View;
import android.view.ViewGroup;

import foo.bar.example.foreretrofit.R;


public class FruitFragment extends Fragment {

    public static FruitFragment newInstance() {
        FruitFragment fragment = new FruitFragment();
        return fragment;
    }

    @Override
    public View onCreateView(LayoutInflater inflater, ViewGroup container, Bundle savedInstanceState) {
        return inflater.inflate(R.layout.fragment_fruit, null);
    }

}

File: example04retrofitcoroutine/src/main/java/foo/bar/example/foreretrofitcoroutine/ui/fruit/FruitFragment.kt
package foo.bar.example.foreretrofitcoroutine.ui.fruit

import android.os.Bundle
import android.view.LayoutInflater
import android.view.View
import android.view.ViewGroup
import androidx.fragment.app.Fragment
import foo.bar.example.foreretrofit.R


class FruitFragment : Fragment() {

    override fun onCreateView(inflater: LayoutInflater, container: ViewGroup?, savedInstanceState: Bundle?): View? {
        return inflater.inflate(R.layout.fragment_fruit, null)
    }

    companion object {
        fun newInstance(): FruitFragment {
            return FruitFragment()
        }
    }

}
